### PR TITLE
Migrate appc/acbuild -> containers/build

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Comment by dustymabe** [comment]
Needed to go from appc/acbuild -> containers/build in order to build the binary. I tried to fix it.

**Comment by dgonyeo** [comment]
+1 on this, I don't appear to have write access to this repo though? @jonboulle

**Comment by jonboulle** [comment]
@dgonyeo your invitation is still pending - you'll need to go to https://github.com/containers and accept it

File: build [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 # https://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-ORG_PATH="github.com/appc"
-REPO_PATH="${ORG_PATH}/acbuild"
+ORG_PATH="github.com/containers"
+REPO_PATH="${ORG_PATH}/build"
 VERSION=$(cd "${DIR}" && git describe --dirty)
-GLDFLAGS="-X github.com/appc/acbuild/lib.Version=${VERSION}"
+GLDFLAGS="-X github.com/${ORG_PATH}/${REPO_PATH}/lib.Version=${VERSION}"
```
Comment (jonboulle) [inline build]: isn't this doubling up? github.com/containers/containers/build

Comment (cdrage) [inline build]: looks like it does:
```
+ GLDFLAGS='-X github.com/github.com/containers/github.com/containers/build/lib.Version=v0.4.0-6-g341b18e'
```

@dustymabe ?

Comment (dustymabe) [inline build]: will fix

Comment (dustymabe) [inline build]: should be fixed now

**Pull request merged**